Login/src/Components/Main/Main.js

```diff
@@ -4,25 +4,26 @@ import Registro from '../Registro/Registro';
 import Navigation from '../Navigation/Navigation';
 import {
     BrowserRouter as Router,
-    Switch,
-    Route,
-    Redirect
+    Routes, /* ya no es switch */
+    Route
 } from "react-router-dom";
 
 export const Main = () => {
   return (
       <>
       <Router>
         <Navigation />
-        <Switch>
-            <Route path="/login">
-                <Login />
-            </Route>
-            <Route path="/registro">
-                <Registro />
-            </Route>
-            <Redirect from="/" to="login" />
-        </Switch>  
+        <Routes>
+        <Route path="/registro"
+                element={<Registro />}
+            />
+            <Route path="/login"
+               element={ <Login />}
+            />
+            <Route path="/"
+               element={ <Login />}
```

Comment:
En este caso no se redirigiría por defecto a login, falta colocar el equivalente a "Redirects", en este caso en la v6 sería:
`    <Route
        path="*"
        element={<Navigate to="/" replace />}
    />`